tenprintcover.py

```diff
@@ -305,7 +310,11 @@ def processColors():
         counts = len(title) + len(author)
         color_seed = int(_map(_clip(counts, 2, 80), 2, 80, 10, 360))
         shape_color = Image.colorHSB(color_seed, base_saturation, base_brightness-(counts % 20))
-        base_color = Image.colorHSB((color_seed + color_distance) % 360, base_saturation, base_brightness)
+        base_color = Image.colorHSB((
```

Comment:
The second `(` belongs to the arithmetic expression `(color_seed + color_distance) % 360` and should go onto the new line with the expression; it’s not a tuple. Pylint didn’t distinguish that?